MonopolyKata/Monopoly.cs

```diff
@@ -76,14 +84,44 @@ private void DetermineOrder()
             }
         }
 
-        public void RunMonopoly()
+        public void RunMonopoly(Int32 numberOfRounds)
         {
             var i = 0;
-            while (i < 20)
+            while (i < numberOfRounds)
             {
                 var j = 1;
-                while (j < 5){
-                    order[j].RollDicePair();
+                while (j < 5)
+                {
+                    //var MoneyTransfer = 0;
+                    //roll dice
+                    order[j].RollDicePair(gameBoard);
+                    var currentLocation = order[j].currentLocation;
+                    order[j].BasicAccountTransfers(currentLocation, gameBoard);
+                    //buy properties
+                    switch (gameBoard.GetStatus(currentLocation))
```

Comment:
if you are naming blocks of code using comments, that is a good place you could refactor into a method to be more descriptive.